bin/spice/pkg/github/runtime_release.go

```diff
@@ -110,12 +110,12 @@ func get_ai_accelerator() (string, bool) {
 	}
 
 	if runtime.GOOS == "linux" {
-		hasCuda, err := has_cuda_device()
+		version, err := get_cuda_version()
```

Comment:
Let's spend a timeboxed amount of time trying to get the fat binary working before we merge this approach of having separate binaries per compute capability